snooty/parser.py

```diff
@@ -1093,13 +1094,19 @@ def __init__(
     ) -> None:
         root = root.resolve(strict=True)
         self.config, config_diagnostics = ProjectConfig.open(root)
-        self.targets = TargetDatabase.load(self.config)
+        self.targets, failed_intersphinx_downloads = TargetDatabase.load(self.config)
```

Comment:
[nit] This value is referred to as `failed_requests` elsewhere, so maybe preferable to keep it generic? But totally your call!